component/loki/source/internal/kafkatarget/target_syncer_test.go

```diff
@@ -66,7 +66,7 @@ func Test_TopicDiscovery(t *testing.T) {
 		}
 		group.mut.Unlock()
 		return reflect.DeepEqual([]string{"topic1"}, group.GetTopics())
-	}, 200*time.Millisecond, time.Millisecond, "expected topics: %v, got: %v", []string{"topic1"}, group.GetTopics())
+	}, time.Second, time.Millisecond, "expected topics: %v, got: %v", []string{"topic1"}, group.GetTopics())
```

Comment:
This test failed on Windows only in CI. I've guessed that maybe timeout is too tight for Windows CI agent and increased it. It passed. If someone has more context, I'd be happy to hear about it.